src/app.module.ts

```diff
@@ -15,11 +18,11 @@ import * as process from 'process';
       username: process.env.USERNAME,
       password: process.env.PASSWORD,
       database: process.env.DATABASE,
-      entities: [],
+      entities: [CountryEntity],
```

Comment:
In order to keep the code clean and well organized, the best practice is to organize your code in modules and only register the modules in AppModule rather than registering every part of sub-modules directly in AppModule.

Suggested solution: create a countries module and only register the countries module within the app module.

Resource: https://docs.nestjs.com/modules

Reply:
The entity is still here, it should be in the `CountriesModule`